cms-lite/src/main/resources/webapp/partials/widgets/stream-resource-modal.html

```diff
@@ -9,7 +9,7 @@ <h4 class="modal-title" id="streamResourceLabel">{{msg('cmslite.header.resource'
             </div>
             <div class="modal-body">
                 <form id="streamResourceForm" name="streamResourceForm" class="form-horizontal"
-                      action="../cmsliteapi/resource/stream/{{select.language}}/{{select.name}}"
+                      action="{{ '../cmsliteapi/resource/stream/' + select.language + '/' + select.name + '/' }}"
```

Comment:
There was no slash at the end of this endpoint in the previous version. Why do we need one now?